src/librbd/cache/pwl/ssd/Types.h

```diff
@@ -6,6 +6,7 @@
   
 #include "acconfig.h"
     
+#include "common/Formatter.h"
```

Comment:
It seems like the preference is to forward-declare `Formatter` in header files, although it's not consistent.

Reply:
Agree, but in this case, code in the header dereferences this class, so an incomplete definition is not enough.
My work tries to move such code out of headers into .cc files, but this header hasn't seen this yet, and this PR is only about adding missing headers, not about other kinds of refactoring.

Reply:
Such code exists in .cc files only since my PR https://github.com/ceph/ceph/pull/62915
And forward declarations were forbidden until my PR https://github.com/ceph/ceph/pull/60331
Just fyi ;-)